src/modules/UI/Wallets/action.js

```diff
@@ -187,10 +195,13 @@ export const editCustomToken = (walletId: string, tokenObj: any, oldCurrencyCode
             coreWalletsToUpdate,
             code: tokenObj.currencyCode
           }))
-          Actions.pop()
+          Actions.waletList()
         })
       })
-      .catch((e) => dispatch(displayErrorAlert(e)))
+      .catch((e) => {
+        dispatch(displayErrorAlert(e.message))
+        dispatch(editCustomTokenFailure())
```

Comment:
This dispatch does not appear to be received by any reducer

Reply:
Ah, looks like I missed that one.